WHO.js

```diff
@@ -0,0 +1,192 @@
+function node(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem:null;}
+
+function detectWeb(doc, url) {
+	if (url.includes('/simple-search')) {
+		return "multiple";
+	}
+	
+	if (text(doc, "h2 > small") == "Collection home page") {
+		return "multiple";
+	}
+	
+	return "document";
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	
+	var rows = doc.querySelectorAll('a.list-group-item');
+	for (let i=0; i<rows.length; i++) {
+		let href = rows[i].href;
+		Z.debug(href);
+		var title = text(rows[i], 'div.row > div.col-md-10 > h5');
+		Z.debug(title);
+		if (!href || !title) continue;
+		if (checkOnly) return;
+		found = true;
+		items[href] = title;
+	}
+	
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var type = detectWeb(doc, url);
+	var item = new Zotero.Item(type);
+	
+	var rows = doc.querySelectorAll("table.itemDisplayTable > tbody > tr");
+	for (let i=0;i<rows.length;i++) {
+		let key = text(rows[i], "td.metadataFieldLabel").trim();
+		let value = text(rows[i], "td.metadataFieldValue");
+		
+		Z.debug(key + " " + value);
+		
+		switch (key) {
+			case 'Title:':
+				item.title = value;
+				break;
+			case 'Language:':
+				let languages = getBrValues(node(rows[i], "td.metadataFieldValue"));
+				languages.forEach(function (value) {
+					if (item.language === undefined) {
+						item.language = value;
+					} else {
+						item.language += ', ' + value;
+					}
+				});
+				break;
+			case 'Authors:':
+				var authors = rows[i].querySelectorAll("a");
+				for (let author of authors) {
+					item.creators.push({lastName: author.textContent,
+										type: "author",
+										fieldMode: 1
+										}
+					);
+				}
+				break;
+			case 'Publisher:':
+				item.publisher = value;
+				break;
+			case 'URI:':
+				item.url = value;
+				break;
+			case 'Abstract:':
+				item.abstractNode = value;
+				break;
+		}
+	}
+	
+	//item.title = "dummy";
+	
+	item.complete();
+}
+
+
+function getBrValues(element) {
+	var child = element.firstChild;
+	var values = [];
+	while (child !== null) {
+		if (child.nodeType == 3) { // Text
+			values.push(child.textContent);
+		}
+		child = child.nextSibling;
+	}
+	return values;
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://apps.who.int/iris/handle/10665/70863",
+		"items": [
+			{
+				"itemType": "document",
```

Comment:
As per above, I want this as a report and then we can probably add some information under report number